### PR TITLE
Manual merge

### PR DESCRIPTION
**Pull request by greg0ire**
Don't merge, just approve

### CODE REVIEW

File: src/Block/Service/EmptyBlockService.php [hunk 1]
```diff
@@ -18,22 +18,11 @@
 use Sonata\BlockBundle\Model\BlockInterface;
 use Sonata\CoreBundle\Validator\ErrorElement;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class EmptyBlockService extends AbstractBlockService
 {
     // NEXT_MAJOR: Remove this method
```
Comment (jordisala1991) [inline src/Block/Service/EmptyBlockService.php]: We did remove some method here on 3.x but not removed the comment (on 3.x is also not removed, so don't do it here either). Just a reminder

File: src/Block/Service/EmptyBlockService.php [hunk 2]
```diff
@@ -18,22 +18,11 @@
...
 
-    /**
-     * {@inheritdoc}
-     */
-    public function configureSettings(OptionsResolver $resolver): void
```
Comment (jordisala1991) [inline src/Block/Service/EmptyBlockService.php]: I am trying to find where this was removed on 3.x, can't really find. Can you show me?

Comment (greg0ire) [reply src/Block/Service/EmptyBlockService.php]: That's probably an error on my part

File: src/Menu/MenuBuilderInterface.php [hunk 3]
```diff
@@ -19,7 +19,7 @@
 /**
  * @author Christian Gripp <mail@core23.de>
  *
- * @deprecated since 3.x, to be removed with 4.0.
+ * @deprecated since 3.9, to be removed with 4.0.
```
Comment (greg0ire) [inline src/Menu/MenuBuilderInterface.php]: I don't know why this changed, but It doesn't matter I guess, since this will be removed in the end.

**Comment by jordisala1991** [comment]
This one was hard to review because one commit was not on 3.x

**Pull request merged**